test/run-test.ps1

```diff
@@ -1,169 +1,62 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
 [cmdletbinding()]
 param(
-    [switch]$UseImageCache,
+    [string]$DotnetInstallDir,
     [string]$Filter,
     [string]$Architecture
 )
 
-Set-StrictMode -Version Latest
-$ErrorActionPreference = 'Stop'
-
-function Exec([scriptblock]$cmd, [string]$errorMessage = "Error executing command: " + $cmd) {
-    & $cmd
-    if ($LastExitCode -ne 0) {
-        throw $errorMessage
-    }
-}
-
-function Get-ActivePlatformImages([PSCustomObject]$manifestRepo, [string]$activeOS) {
-    return $manifestRepo.Images |
-        ForEach-Object { $_.Platforms } |
-        Where-Object { $_.os -eq "$activeOS" } |
-        Where-Object { ( [string]::IsNullOrEmpty($Architecture) -and -not [bool]($_.PSobject.Properties.name -match "architecture"))`
-            -or ( [bool]($_.PSobject.Properties.name -match "architecture") -and $_.architecture -eq "$Architecture" ) }
-}
+$DotnetInstallDir = "$PSScriptRoot/../.dotnet"
```

Comment:
Why is `DotnetInstallDir` a parameter when it gets initialized this way?

Reply:
An oversight on my part - removing parameter.